build.sh

```diff
@@ -8,4 +8,4 @@ while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symli
 done
 ScriptRoot="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
-. "$ScriptRoot/build/build.sh" --build --restore --log "$@"
+. "$ScriptRoot/build/build.sh" --build --restore "$@"
```

Comment:
Why was the log command removed?

Reply:
It is not a valid arg in the new version of the repo toolset and caused source-build builds to fail.

Reply:
The build now always produces binlog.

---
In reply to: [184189056](https://github.com/dotnet/sdk/pull/2178#discussion_r184189056) [](ancestors = 184189056)